website/docs/docs/cloud/manage-access/self-service-permissions.md

```diff
@@ -1,42 +1,81 @@
+- **IT** &mdash; Partial access to the account settings including users, integrations, billing, and API settings. Cannot create or edit connects or access the project at all. Functions the same regardless of group assignments.
 
+See [Seats and Users](/docs/cloud/manage-access/seats-and-users) for more information on the impact of licenses on these permissions.
 
-### Prerequisites
+## Table of groups, licenses, and permissions
 
-You need to create an Account Admins group before removing any other groups.
+Key:
 
-1. Create an Account Admins group.  
-2. Assign at least one user to the Account Admins group. The assigned user can manage future group, SSO mapping, and user or group assignment.
+* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `read`.
+* (R)ead &mdash; Can view but can not create or change any fields.
+* No value &mdash; No access to the feature.
 
-### Remove the Owner and Member groups
+Permissions:
 
-Follow these steps for both Owner and Member groups:
+* Account-level permissions &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.
+* Project-level permissions &mdash; Permissions related to the projects in dbt Cloud. For example, Explorer and the IDE.
+
+The following tables outline the access that users have if they are assigned a Developer license and the Owner or Member group, Read-only license, or IT license.
+
+#### Account permissions for account roles
+
+| Account-level permission| Owner | Member | Read-only license|  IT license  |
+|:------------------------|:-----:|:------:|:----------------:|:------------:|
+| Account settings        |   W   |   W    |                  |       W      |
+| Billing                 |   W   |        |                  |       W      |
+| Invitations             |   W   |   W    |                  |       W      |
+| Licenses                |   W   |   R    |                  |       W      |
+| Users                   |   W   |   R    |                  |       W      |
+| Project (create)        |   W   |   W    |                  |       W      |
+| Connections             |   W   |   W    |                  |       W      |
+| Service tokens          |   W   |        |                  |       W      |
+| Webhooks                |   W   |   W    |                  |              |
+ 
+#### Project permissions for account roles
+
+|Project-level permission | Owner | Member  | Read-only | IT license |
+|:------------------------|:-----:|:-------:|:---------:|:----------:|
+| Adapters                |   W   |    W    |    R      |             |
+| Connections             |   W   |    W    |    R      |             |
+| Credentials             |   W   |    W    |    R      |             |
+| Custom env. variables   |   W   |    W    |    R      |             |
+| dbt adapters            |   W   |    W    |           |             |
```

Comment:
what does dbt adapters mea? how is it diff to connections?

Reply:
Good catch. Removing that